app/models/propertyrepresentation/AddAgentOptions.scala

```diff
@@ -18,34 +18,34 @@ package models.propertyrepresentation
 
 import models.{EnumFormat, NamedEnum, NamedEnumSupport}
 
-sealed trait ManagePropertiesOptions extends NamedEnum {
+sealed trait AddAgentOptions extends NamedEnum {
```

Comment:
Shouldn't we be using Enumeration?

Reply:
NamedEnum seem to be the standard in voa-property-linking-frontend

Reply:
Sometimes things need to change.

Reply:
im happy to change it if you can let me know what we will gain from making that change. If its just a case of preference, then I will keep it as it is

Reply:
It's about standards and patterns. I'm confused when I see different approaches to doing the same thing. 
Here I am asking a question "Shouldn't we be using Enumeration?" - not telling you to change it. Also, this has been raised before in PR and I seem to remember Dan saying we should use Enumeration?

Reply:
Maybe some of the discussion points here could be part of Dan's handover?